app/controllers/application_controller.rb

```diff
@@ -2,4 +2,9 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  
+  def show_player(player)
+    Player.find(player).email
```

Comment:
With an expression as small as this you could just add this line to the layout and avoid the creation of a helper method.